### PR TITLE
Billybratz/add loading component

### PR DESCRIPTION
**Pull request by reharri7**
@wbratz

### CODE REVIEW

File: src/app/pages/loading/loading.module.ts [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
     CommonModule,
     FormsModule,
     IonicModule,
-    LoadingPageRoutingModule
+    RouterModule.forChild([
+      {
+        path: '',
+        component: LoadingPage,
+      },
+    ]),
```
Comment (reharri7) [inline src/app/pages/loading/loading.module.ts]: What is this? Doesn't this belong in the router module?

Comment (reharri7) [reply src/app/pages/loading/loading.module.ts]: @wbratz ^^

**Review by wbratz — APPROVED** [review]

File: src/app/pages/loading/loading.module.ts (hunk 1 above)
Comment (wbratz) [reply src/app/pages/loading/loading.module.ts]: Oh, maybe this was probably just me trying to figure out why I couldn't navigate, I'll address.

Comment (wbratz) [reply src/app/pages/loading/loading.module.ts]: reverted changes, tested, everything still works.

**Review by reharri7 — COMMENTED** [review]
Dope

**Pull request merged**